src/main/java/org/saltyrtc/client/signaling/InitiatorSignaling.java

```diff
@@ -599,7 +599,7 @@ Peer getPeerWithId(short id) throws SignalingException {
             return this.server;
         } else if (this.isResponderId(id)) {
             //noinspection ConstantConditions
-            if (this.getState() == SignalingState.TASK && this.responder != null & this.responder.getId() == id) {
+            if (this.getState() == SignalingState.TASK && this.responder != null && this.responder.getId() == id) {
```

Comment:
Oh, that's a nice catch! Especially since the noinspection comment above silences the warning in IntelliJ :face_with_head_bandage: Fortunately it doesn't change anything about the logic since both sides are booleans.

Can you also remove the noinspection comment above?